components/esp_websocket_client/esp_websocket_client.c

```diff
@@ -759,6 +759,22 @@ esp_err_t esp_websocket_client_stop(esp_websocket_client_handle_t client)
     return ESP_OK;
 }
 
+esp_err_t esp_websocket_client_setPingInterval(esp_websocket_client_handle_t client, size_t ping_interval_sec)
+{
+    if (client == NULL) {
+        return ESP_ERR_INVALID_ARG;
+    }
+    if (client->config == NULL) {
+        return ESP_ERR_INVALID_ARG;
+    }
+    if (ping_interval_sec == 0) {
+        return ESP_ERR_INVALID_ARG;
+    }
```

Comment:
Why the extra verbosity? `if ( !( client && client->config && ping_interval_sec ) )` or some other single if clause variant would suffice.